main.js

```diff
@@ -0,0 +1,149 @@
+var VideoModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      videoId: "",
+      videoThumbnamil: "",
+      videoTitle: "",
+      videoDesc: "",
+    };
+  },
+});
+
+// Videos collection - add API fetch here
+var VideosCollection = Backbone.Collection.extend({ 
+  url: function (topic) {
+    console.log(topic);
+    if (!topic) {
+      return "https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=cercle&type=video&videoEmbeddable=true&key=AIzaSyCjo4u-wcr_ExFNPxiYlWZP3LLr-ythijE";
+    } else {
+      return (
+        "https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=5&q=" +
+        topic +
+        "cercle&type=video&videoEmbeddable=true&key=AIzaSyCjo4u-wcr_ExFNPxiYlWZP3LLr-ythijE"
+      );
+    }
+  },
+
+  model: VideoModel,
+
+  initialize: function () {
+    this.on("add", function (model) {
+      model.fetch();
+    });
+  },
+  
+  parse: function (response) {
+    return response.items.map(function (x) {
+      return {
+        videoId: x.id.videoId,
+        videoThumbnail: x.snippet.thumbnails.default.url,
+        videoTitle: x.snippet.title,
+        videoDesc: x.snippet.description,
+      }
+    })
+  }
+});
+
+var AppModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      videos: new VideosCollection(),
+    };
+  }
+});
+
+var VideoView = Backbone.View.extend({
+  className: 'video',  // to reference later
+
+  template: Handlebars.compile($('#five-display-template').html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  },
+});
+
+var MainVideoView = Backbone.View.extend({
+  className: 'mainVideo',
+
+  template: Handlebars.compile($('#main-display-template').html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  }
+})
+
+var AppView = Backbone.View.extend({
+  el: $("body"),
+
+  events: {
+    "click .search": "newSearch", 
+    "click .click-here": "newMain"
+  },
+  
+  initialize: function () {
+    this.$search = this.$('#search');
+    this.listenTo(this.model.get('videos'), 'change', this.renderVideo);
```

Comment:
You could refactor to:

```js
this.listenTo(this.model.get('videos'), 'change', function () {
  this.renderVideo();
  this.renderMainVideo();
});
```

Any time you're repeating yourself, there is a way to refactor.